test/clj_diff/test/custom_comparator.clj

```diff
@@ -0,0 +1,71 @@
+(ns clj-diff.test.custom-comparator
+  (:use [clj-diff.core] :reload)
+  (:use [clojure.test]))
```

Comment:
this seems a bit unusual, but I see this is in line with how other tests are doing things, e.g.
https://github.com/droitfintech/clj-diff/blob/1301f9aaf4ec8cda585b6fd0bfdda965853368fc/test/clj_diff/test/core.clj#L1-L3

Reply:
You're right. corrected.